test/test_deprecations.jl

```diff
@@ -40,90 +40,88 @@ end
+    @test_throws Ferrite.DeprecationError test_combo(CellValues, 3, RefCube, (:legendre, 1), Lagrange{RefHexahedron, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 0, RefCube, (1,), Lagrange{RefLine, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 0, RefCube, (:legendre, 1), Lagrange{RefLine, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 1, RefCube, (1,), Lagrange{RefQuadrilateral, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 1, RefCube, (1,), Lagrange{RefQuadrilateral, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 1, RefCube, (:legendre, 1), Lagrange{RefQuadrilateral, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 1, RefCube, (:legendre, 1), Lagrange{RefQuadrilateral, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 2, RefCube, (1,), Lagrange{RefHexahedron, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 2, RefCube, (1,), Lagrange{RefHexahedron, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 2, RefCube, (:legendre, 1), Lagrange{RefHexahedron, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 2, RefCube, (:legendre, 1), Lagrange{RefHexahedron, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 1, RefTetrahedron, (1,), Lagrange{RefTriangle, 1}())
+    @test_throws Ferrite.DeprecationError test_combo(FacetValues, 1, RefTetrahedron, (:legendre, 1), Lagrange{RefTriangle, 1}())
 end
 
 @testset "Ferrite.value and Ferrite.derivative" begin
     ip = Lagrange{RefQuadrilateral, 1}()
     ξ = zero(Vec{2})
-    @test (@test_deprecated Ferrite.value(ip, ξ)) == [shape_value(ip, ξ, i) for i in 1:getnbasefunctions(ip)]
-    @test (@test_deprecated Ferrite.derivative(ip, ξ)) == [shape_gradient(ip, ξ, i) for i in 1:getnbasefunctions(ip)]
-    @test (@test_deprecated Ferrite.value(ip, 1, ξ)) == shape_value(ip, ξ, 1)
+    @test_throws Ferrite.DeprecationError Ferrite.value(ip, ξ)
+    @test_throws Ferrite.DeprecationError Ferrite.derivative(ip, ξ)
+    @test_throws Ferrite.DeprecationError Ferrite.value(ip, 1, ξ)
 end
 
 @testset "facesets" begin
-grid = generate_grid(Quadrilateral, (2,2))
-addfaceset!(grid, "right_face", x -> x[1] ≈ 1)
-@test getfacetset(grid, "right_face") == getfacetset(grid, "right")
-addfaceset!(grid, "right_face_explicit", Set(Ferrite.FaceIndex(fi[1], fi[2]) for fi in getfacetset(grid, "right")))
-@test getfacetset(grid, "right_face_explicit") == getfacetset(grid, "right")
+    grid = generate_grid(Quadrilateral, (2,2))
+    @test_throws Ferrite.DeprecationError addfaceset!(grid, "right_face", x -> x[1] ≈ 1)
+    @test_throws Ferrite.DeprecationError addfaceset!(grid, "right_face_explicit", Set(Ferrite.FaceIndex(fi[1], fi[2]) for fi in getfacetset(grid, "right")))
```

Comment:
@KnutAM this is supposed to be deprecated? I guess we removed all methods of `addfaceset!`?

Reply:
Yes, but done with `@warn ... maxlog=1` to actually show the warning for normal users not running tests. 
https://github.com/Ferrite-FEM/Ferrite.jl/blob/900f6ed3547ce5d70e0ecdc998c9fb2f81aaeb94/src/deprecations.jl#L356-L366
But if we error, they will see it too so that is also fine by me (I think this way, it was possible to hack around to get e.g. FerriteGmsh working, but this is no longer required after updating FerriteGmsh)